### DISCUSSION TITLE
vignettes in imagemagick?

### DISCUSSION THREAD

**Discussion by bruce-simonson** [ImageMagick/ImageMagick#6067] [Help] [answered]
### ImageMagick version

7.1.0-61

### Operating system, version and so on

Windows 10

### Description

I’d like to apply a “vignetting” overlay to an image. This is possible on an iphone, using the default photo editing app.

Vignettes (as implemented by Apple), are “dodges” from the corners, usually done in black, to darken and soften the corners, and draw attention to what’s in the middle of the image. (Apple’s app also allows for lightening vignettes, where the corners are lightened, instead of darkened).

The vignetted areas are semi-transparent, so you can still see what’s in the image, underneath the vignette.

I can’t find or figure out how to do this with imagemagick, but I’ll bet its possible.

I’ve prepared a pdf, which I’ll try to attach. If it has to be a jpg or similar, I’m not sure that the detail will come through. I’ll give it a try.

Note that the underlying image isn’t necessarily square, but it is rectangular.

Here’s the jpg version of the vignette examples:

![iphone vignette example (jpg)](https://user-images.githubusercontent.com/66903636/219011334-f36c4ecb-c58c-4a5c-8f9f-ce99b24f8ae5.jpg)

**Comment by snibgo** [comment, marked as answer]
Have you considered `-vignette`? See https://imagemagick.org/script/command-line-options.php#vignette

**Reply by bruce-simonson** [reply]
Thanks, don't know how I missed this in the documentation.  (doh).  

I could try to dissemble, and say that I was fishing for a more complicated way to achieve this.  (doh).

The default color for the vignette is white ... can you help me with the syntax for using black instead?

**Comment by fmw42** [comment]
use -background black for setting the vignette color.

![lena](https://user-images.githubusercontent.com/13458218/219137649-d07af567-e14e-40a8-ac18-36d8a70b34d9.jpg)

`convert lena.png -background black -vignette 0x10+10+10% x.png`

![x](https://user-images.githubusercontent.com/13458218/219137681-4dee730c-f63c-449b-ab25-eecaa42cdfb1.png)
